src/patches/patchSettings.tsx

```diff
@@ -41,6 +41,20 @@ export default function patchSettings() {
                 key: "AliucordErrors",
                 title: "Errors",
                 render: ErrorsPage
+            },
+            // Render custom page dynamically. For options, some props from https://reactnavigation.org/docs/native-stack-navigator#options are valid
+            AliucordCustomPage: {
+                key: "AliucordCustomPage",
+                title: "Aliucord Page",
+                render: ({ render: PageView, ...options }) => {
+                    const navigation = NavigationNative.useNavigation();
+
+                    React.useEffect(() => {
+                        options && navigation.setOptions({ ...options });
+                    }, []);
+
+                    return <PageView />;
+                }
```

Comment:
what does this do?
isnt this testing code

Reply:
```
// Render custom page dynamically
```

Reply:
I'm confused
custom page for what

Reply:
I'm confused
custom page for what